CHANGELOG.md

```diff
@@ -211,6 +212,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Updated `t3c-apply` to reduce mutable state in `TrafficOpsReq` struct.
 - Updated Golang dependencies
 - [#6506](https://github.com/apache/trafficcontrol/pull/6506) - Updated `jackson-databind` and `jackson-annotations` Traffic Router dependencies to version 2.13.1
+- Traffic Router now includes a `Content-Length` header in each response.
```

Comment:
I don't believe this was added in 6.1. Unless you're doing this preemptively for a backport or something?

Reply:
Oops, didn't remove the 6.1.0 changelog entry in 88e2490439, which added a changelog entry to the unreleased section. Removed the 6.1.0 one in 7774d21768.